frontend/src/App.tsx

```diff
@@ -1,31 +1,26 @@
 import { useState } from 'react';
-import reactLogo from './assets/react.svg';
-import viteLogo from '../../../../../vite.svg';
 import './App.css';
+import MockPage from './components/mockPage/MockPage';
+import LoginPage from './components/loginPage/LoginPage';
+
+
+interface IUser {
+	name: string;
+	loggedIn: boolean;
+}
 
 export function App() {
-  const [count, setCount] = useState(0);
 
-  return (
-    <>
-      <div>
-        <a href="https://vitejs.dev" target="_blank" rel="noreferrer">
-          <img src={viteLogo} className="logo" alt="Vite logo" />
-        </a>
-        <a href="https://react.dev" target="_blank" rel="noreferrer">
-          <img src={reactLogo} className="logo react" alt="React logo" />
-        </a>
-      </div>
-      <h1>Sportify Me!</h1>
-      <div className="card">
-        <button type="button" onClick={() => setCount((number) => number + 1)}>
-          count is {count}
-        </button>
-        <p>
-          Edit <code>src/App.tsx</code> and save to test HMR
-        </p>
-      </div>
-      <p className="read-the-docs">Click on the Vite and React logos to learn more</p>
-    </>
-  );
+	const noNameUser: IUser = {
```

Comment:
That's probably not the best way, I think we should use session cookies for logging in.

Reply:
I have no experience with session cookies, but hey I'm here to learn! This was just a mockup, anyway.

Reply:
Yeap, that's the whole point! :) Let's learn and improve our skills together 💪